apps/labrinth/src/auth/email/mod.rs

```diff
@@ -34,9 +35,20 @@ pub fn send_email_raw(
     let username = dotenvy::var("SMTP_USERNAME")?;
     let password = dotenvy::var("SMTP_PASSWORD")?;
     let host = dotenvy::var("SMTP_HOST")?;
+    let port = dotenvy::var("SMTP_PORT")?.parse::<u16>().unwrap_or(25);
```

Comment:
Let's change the default value of `SMTP_PORT` to match `lettre`'s SMTP over TLS `SUBMISSIONS_PORT`, which is 465, to minimize changes of production deployments breaking due this new environment variable being unset:

```suggestion
    let port = dotenvy::var("SMTP_PORT")?.parse::<u16>().unwrap_or(465);
```